### PR TITLE
Added a Cypress test for archived articles.

### PR DESCRIPTION
**Pull request by rajputanuj31**
Added a Cypress test for archived articles.

### CODE REVIEW

**Comment by brianteeman** [comment]
This (like many of the others) is a test of the archived menu item and not of the display of archived articles.

Shouldnt this test of the menu type also check that articles in any of the other 3 states are not displayed

**Comment by rajputanuj31** [comment]
> This (like many of the others) is a test of the archived menu item and not of the display of archived articles.
> 
> Shouldnt this test of the menu type also check that articles in any of the other 3 states are not displayed

Other states are handled in 3rd test.

File: tests/System/integration/site/components/com_content/Archived.cy.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        cy.contains('article 1');
+        cy.contains('article 2');
+      });
+  });
+
+  it('can not display not archived articles', () => {
+    cy.db_createMenuItem({ title: 'automated test archive article', link: 'index.php?option=com_content&view=archive' })
+      .then(() => cy.db_createArticle({ title: 'article 1', state: 1 }))
+      .then(() => cy.db_createArticle({ title: 'article 2', state: 0 }))
+      .then(() => cy.db_createArticle({ title: 'article 2', state: 2 }))
```
Comment (laoneo) [inline tests/System/integration/site/components/com_content/Archived.cy.js]:
```suggestion
      .then(() => cy.db_createArticle({ title: 'article 3', state: 2 }))
```

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**